Makefile

```diff
@@ -257,10 +257,15 @@ debian: sdist
 
 deb: deb-src
 	@for DIST in $(DEB_DIST) ; do \
+	    if test $${DIST} = trusty ; then \
+	        CRYPTO_BACKEND=pycrypto ; \
```

Comment:
this is needed for more than trusty, a numeric version selection would be better .. then there is Ubuntu and other debian derivatives ...